### PR TITLE
[android] Chinese translation updated

### PR DESCRIPTION
**Pull request by TheJokerW**
using  Simplified Chinese and more popular words

### CODE REVIEW

**Comment by TheJokerW** [comment]
image
I accidentally used “#975”，and change the commit message

**Review by TheJokerW — COMMENTED** [review]
done

**Comment by gubatron** [comment]
Thank you for this, will review and merge.

File: android/res/values-zh/strings.xml [hunk 1]
```diff
@@ -19,16 +19,16 @@
  */
 -->
 <resources>
-    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[披露： 虽然建议您下载， <b>您不需要VPN使用FrostWire。 </b>]]></string>
-    <string name="tip_touch_tags_to"><![CDATA[ <b>信息:</b> 触摸应用的标签]]></string>
-    <string name="unprotected_connections_visibility_bullet_html"><![CDATA[ <b>您的隐私是一项基本人权</b>. 不受保护的连接意味着您在线进行的所有操作（包括<b>下载，搜索，消息传递</b>）都可以被网络上的任何人监控和拦截，例如： 互联网服务提供商，黑客，政府。<br><br><b>我们强烈建议您使用VPN服务</b>. <br> 以下是一些提供可靠服务并与FrostWire配合良好的选项：]]></string>
+    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[公开的秘密： 虽然建议您下载， <b>您不需要VPN使用FrostWire。 </b>]]></string>
+    <string name="tip_touch_tags_to"><![CDATA[ <b>提示:</b> 触摸以应用标签]]></string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:23]: Seems to me it was better before.

File: android/res/values-zh/strings.xml [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
+    <string name="unprotected_connections_visibility_bullet_html"><![CDATA[ <b>您的隐私权是一项基本人权</b>. 不受保护的连接意味着您在线进行的所有操作（包括<b>下载，搜索，消息传递</b>）都可以被网络上的任何人监控和拦截，例如： 互联网服务提供商，黑客，政府。<br><br><b>我们强烈建议您使用VPN服务</b>. <br> 以下是一些提供可靠服务并与FrostWire配合良好的选项：]]></string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:24]: 👍

File: android/res/values-zh/strings.xml [hunk 3]
```diff
@@ -19,16 +19,16 @@
...
     <string name="protected_connections_visibility_bullet_html"><![CDATA[ 您已使用VPN连接到Internet。 <br><br> 如果您尚未使用，我们建议您使用以下服务之一加密您的连接并<b>保护您的隐私</b>。 ]]></string>
     <string name="activity_preferences_name">设置</string>
     <string name="exit">退出</string>
     <string name="search_label">搜索</string>
     <string name="support_free_software">支持免费软件</string>
-    <string name="waiting">等待?</string>
-    <string name="type_to_filter_results">打字过滤结果</string>
+    <string name="waiting">等待中...</string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:30]: Please use the ellipsis single character here:
`…`

File: android/res/values-zh/strings.xml [hunk 4]
```diff
@@ -19,16 +19,16 @@
...
+    <string name="type_to_filter_results">输入以过滤结果</string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:31]: 👍

File: android/res/values-zh/strings.xml [hunk 5]
```diff
@@ -37,19 +37,19 @@
     <string name="pictures">图片</string>
     <string name="ringtones">铃声</string>
     <string name="documents">文档</string>
-    <string name="share">共享</string>
+    <string name="share">分享</string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:40]: :+1
thanks for catching this one

File: android/res/values-zh/strings.xml (hunk 1 above)
Comment (TheJokerW) [reply android/res/values-zh/strings.xml:23]: sorry about this，good check

File: android/res/values-zh/strings.xml (hunk 3 above)
Comment (TheJokerW) [reply android/res/values-zh/strings.xml:30]: ok

File: android/res/values-zh/strings.xml [hunk 6]
```diff
@@ -37,19 +37,19 @@
...
-    <string name="update_message">FrostWire已为您下载新的绩效考核包，如想现在安装，请按OK键。</string>
+    <string name="update_message">FrostWire已为您下载强力更新包，如想现在安装，请按确定。</string>
     <string name="about">关于</string>
     <string name="send_file_using">发送文件使用</string>
     <string name="my_files">我的文件</string>
     <string name="settings">首选项</string>
     <string name="files">文件</string>
     <string name="minimize_frostwire">FrostWire最小化</string>
-    <string name="are_you_sure_you_wanna_leave">确定想回到主页界面?</string>
+    <string name="are_you_sure_you_wanna_leave">确定想回到主界面?</string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:52]: 👎 Please revert to the old version, it's better.

File: android/res/values-zh/strings.xml [hunk 7]
```diff
@@ -59,8 +59,8 @@
     <string name="other_settings">其他设置</string>
     <string name="vibrate_on_finished_download">下载完成后振动</string>
     <string name="cant_open_file">很抱歉，不知道如何打开此文件.</string>
-    <string name="download_added_to_queue">1个文件开始下载</string>
-    <string name="are_you_sure_delete_files">确定想要删除选定文件?</string>
+    <string name="download_added_to_queue">1个文件将开始下载</string>
+    <string name="are_you_sure_delete_files">确定想要永久删除选定文件?</string>
     <string name="delete_file_menu_action">删除</string>
```
Comment (gubatron) [inline android/res/values-zh/strings.xml:64]: 👍

**Pull request merged**